plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioParquetConnectorTest.java

```diff
@@ -15,7 +15,9 @@
 
 import io.trino.Session;
 import org.apache.iceberg.FileFormat;
+import org.testng.annotations.Test;
 
+@Test(singleThreaded = true)
```

Comment:
What is the reasoning behind this annotation? Could you please add a comment in the code explaining this choice?

Reply:
We have single MinIO instance. I would like to avoid conflicts. But i think separate resources would be sufficient.
Will remove it.

Reply:
Could you try adding the annotation and the comments to the BaseIcebergMinioConnectorTest  (to avoid doing the code duplication). I'm not sure though whether it is being picked up by TestNG